content/blog/2018/08/2018-08-13-simple-pull-request-plugin-final-evaluation.adoc

```diff
@@ -0,0 +1,227 @@
+ args: -v /tmp:/tmp
+
+  tools:
+    maven : maven_3.0.1
+    jdk : jdk8
+
+configuration:
+  # Push PR changes to the target branch if the build succeeds.
+  # default value is false
+  pushPrOnSuccess: false
+
+  # Trusted user to approve pull requests
+  prApprovers:
+    - username1
+    - username2
+    - username3
+
+environment:
+  variables:
+    variable_1: value_1
+    variable_2: value_2
+
+  # Credentials contains only two fields. credentialId must be present in the Jenkins Credentials
+  credentials:
+    - credentialId : fileCredentialId
+      variable : FILE
+
+      # In user scripts Username and Password can be accessed by LOGIN_USR and LOGIN_PSW
+      # respectively as environment variales
+    - credentialId : dummyGitRepo
+      variable : LOGIN
+
+stages:
+  - name: stage1
+    agent: any
+    steps:
+      - sh: "scripts/hello"
+      - sleep:
+          time: 2
+          unit: SECONDS
+      - sleep: 2
+      - junit:
+          testResults: "target/**.xml"
+          allowEmptyResults: true
+          testDataPublishers:
+            - AutomateTestDataPublisher
+            - JunitResultPublisher:
+                urlOverride: "urlOverride"
+    # Post section for "stage1". All Conditions which are available in Jenkins
+    # declarative pipeline are supported
+    post:
+      failure:
+        - sh: "scripts/hello"
+
+# Outer post section. Just like declarative pipeline.
+post:
+  always:
+    - sh: "scripts/hello"
+----
+
+=== Tasks completed in Coding Phase 3
+
+. Add unit tests, JenkinsRule tests link:https://issues.jenkins-ci.org/browse/JENKINS-52495[JENKINS-52495]
+. Refactor snippet generator to extensions (link:https://issues.jenkins-ci.org/browse/JENKINS-52491[JENKINS-52491])
+. Release 1.0 (link:https://issues.jenkins-ci.org/browse/JENKINS-52519[JENKINS-52519])
+. Plugin overview blog post
+
+
+=== Future tasks
+https://issues.jenkins-ci.org/browse/JENKINS-52452[Phase 3 Jira Epic]
+
+. Support the “when” Declarative Pipeline directive (link:https://issues.jenkins-ci.org/browse/JENKINS-52520[JENKINS-52520])
+. Nice2have: Support hierarchical report types (link:https://issues.jenkins-ci.org/browse/JENKINS-52521[JENKINS-52521])
+. Acceptance Test Harness tests link:https://issues.jenkins-ci.org/browse/JENKINS-52496[JENKINS-52496]
+. Automatic Workspace Cleanup when PR is closed (link:https://issues.jenkins-ci.org/browse/JENKINS-51897[JENKINS-51897])
+
+. Test Multi-Branch Pipeline features support:
+.. Support for webhooks (link:https://issues.jenkins-ci.org/browse/JENKINS-51941[JENKINS-51941])
+.. Check if trusted people have approved a pull request and start build accordingly (link:https://issues.jenkins-ci.org/browse/JENKINS-52517[JENKINS-52517])
+. Finalize documentation (link:https://issues.jenkins-ci.org/browse/JENKINS-52518[JENKINS-52518])
+
+. Test the integration with various platforms Bitbucket, Gitlab, Github.
+
+
+=== Phase 3 evaluation presentation video
+
+
+Video:
+++++
+<center>
```

Comment:
You should add a direct link too, because this didn't render when I tried viewing just now (same with the slides)